doc/src/manual/metaprogramming.md

```diff
@@ -263,29 +263,28 @@ julia> eval(ans)
 julia> ex = :(a + b)
 :(a + b)
 
-julia> eval(ex)
+julia> eval(current_module(), ex)
```

Comment:
If that gets resolved then the closest-candidates from https://github.com/JuliaLang/julia/pull/20224#discussion_r98220018 can probably be brought back too